src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +56,88 @@ void createStation() {
+                        .body(params)
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().post("/stations")
+                        .then().log().all()
+                        .extract();
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        params.clear();
+        params.put("name", "신사역");
+        response =
+                RestAssured.given().log().all()
+                        .body(params)
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().post("/stations")
+                        .then().log().all()
+                        .extract();
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        //when
+        response =
+                RestAssured.given().log().all()
+                        .when().get("/stations")
+                        .then().log().all()
+                        .extract();
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+        //then
+        int TestingNumber = 2;
+        List<String> stationsList =
+                RestAssured.given().log().all()
+                        .when().get("/stations")
+                        .then().log().all()
+                        .extract().jsonPath().getList("name", String.class);
+        assertThat(stationsList.size()).isEqualTo(TestingNumber);
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    // testing: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 제거한다.")
+    @Test
+    void 지하철역_제거_테스트() {
+        //given
+        Map<String, String> params = new HashMap<>();
+        params.put("name", "부산역");
+        ExtractableResponse<Response> response =
+                RestAssured.given().log().all()
+                        .body(params)
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().post("/stations")
+                        .then().log().all()
+                        .extract();
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        //when
+        String id = response.jsonPath().getString("id");
+        response =
+                RestAssured.given().log().all()
+                        .when().delete("/stations/" + id)
+                        .then().log().all()
+                        .extract();
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+
+        //then
+        int TestingNumber = 0;
+        List<String> stationsList =
+                RestAssured.given().log().all()
+                        .when().get("/stations")
+                        .then().log().all()
+                        .extract().jsonPath().getList("name", String.class);
+        assertThat(stationsList.size()).isEqualTo(TestingNumber);
```

Comment:
`isEmpty()`를 활용해보시는 것도 좋을 것 같습니다.
```suggestion
        assertThat(stationsList).isEmpty();
```